src/components/approval-type/ApprovalType.vue

```diff
@@ -8,87 +11,81 @@
       >
         <label :class="{ 'error-text': invalidSection }">Approval Type</label>
       </v-col>
+
       <v-col
         cols="12"
         sm="9"
-        class="mt-n4"
+        class="mb-n5"
       >
         <v-radio-group
-          v-model="approvalTypeSelected"
-          class="payment-group pt-0"
-          @change="radioButtonChanged"
+          class="payment-group mt-0"
+          :value="approvalTypeSelected"
+          @change="setApprovalTypeSelected($event)"
         >
-          <!-- COURT ORDER section -->
-          <template v-if="!isCourtOrderRadio">
-            <span class="v-label ml-2 mb-2">{{ getRadioText(ApprovalTypes.VIA_COURT_ORDER) }}</span>
-          </template>
-          <template v-else>
-            <v-radio
-              id="court-order-radio"
-              class="mb-n3"
-              :label="getRadioText(ApprovalTypes.VIA_COURT_ORDER)"
-              :value="ApprovalTypes.VIA_COURT_ORDER"
-            />
-          </template>
-          <v-form
-            id="court-num-form"
-            ref="courtNumRef"
-            v-model="valid"
-            class="mt-8 ml-2"
+          <!-- court order section -->
+          <v-radio
+            v-if="isCourtOrderRadio"
+            id="court-order-radio"
+            class="mb-0"
+            :label="getRadioText(ApprovalTypes.VIA_COURT_ORDER)"
+            :value="ApprovalTypes.VIA_COURT_ORDER"
+          />
+          <span
+            v-else
+            class="v-label"
           >
-            <v-expand-transition class="pb-0 mb-0">
+            {{ getRadioText(ApprovalTypes.VIA_COURT_ORDER) }}
+          </span>
+          <v-form ref="courtNumRef">
+            <v-expand-transition>
               <v-text-field
                 v-if="approvalTypeSelected === ApprovalTypes.VIA_COURT_ORDER"
                 id="court-order-number-input"
-                v-model="courtOrderNumberText"
+                class="mt-4"
                 label="Court Order Number"
-                :rules="courtOrderNumRules"
+                :value="courtOrderNumber"
+                :rules="validate ? courtOrderNumRules : []"
```

Comment:
The rules are conditional on the validate prop (but you have to wait a tick for validation to happen).